prow/plugins/trigger/trigger.go

```diff
@@ -254,6 +254,15 @@ func TrustedUser(ghc trustedUserClient, onlyOrgMembers bool, trustedOrg, user, o
 		}
 	}
 
+	// Determine if user is on trusted_apps list.
+	// This allows automatic tests execution for GitHub automations that cannot be added as collaborators.
+	for _, trustedApp := range trustedApps {
+		// username must explicitly end with [bot] suffix
+		if strings.HasSuffix(user, "[bot]") && user == trustedApp {
```

Comment:
Could we instead ignore users that do not have the suffix and strip the suffix prior to comparing, so that ppl do not have to put it in their config? That would be a lot nicer to use.

Reply:
@alvaroaleman addressed your review comment. At first I wanted to have the usernames explicitly defined with [bot] suffix, to easily distinct GitHub apps from other users, but since the field will be unly used for bots it sure will be easier to use it that way.